features/step_definitions/edit_review_steps.rb

```diff
@@ -0,0 +1,52 @@
+When(/^I attempt to edit any review$/)do
+  new_user = User.new
+  new_user.email = "user1@tamu.edu"
+  new_user.password_digest = "12345"
+  new_user.save
+  current_user = User.where(:email => 'user1@tamu.edu').first
```

Comment:
My only worry with this is if there is something special that happens when you actually log in.  If there isn't then it should be fine. @dfields16